examples/gpt-2/README.md

```diff
@@ -145,89 +159,95 @@ This section shows how we can fine-tune the pre-trained GPT2 model and use the r
 
 We first preprocess data with the GPT-2 BPE encoding. 
 
-A toy dataset is provided under [`data/toy/`](data/toy) which includes `train.txt`, `dev.txt`, and `test.txt`. This example will fit the GPT-2 model on `train.txt`, evaluate perplexity on `dev.txt`, and do continuation generation using `test.txt` as the context.
+A toy dataset is provided under [`data/toy/`](data/toy) which includes `train.txt`, `dev.txt`, and `test.txt`. This
+example will fit the GPT-2 model on `train.txt`, evaluate perplexity on `dev.txt`, and do continuation generation using
+`test.txt` as the context.
 
-Run the following cmd to transform the data into [pickle](https://docs.python.org/3/library/pickle.html) format and perform processing such as truncation, BPE encoding, adding special tokens, etc:
+Run the following cmd to transform the data into [pickle](https://docs.python.org/3/library/pickle.html) format and
+perform processing such as truncation, BPE encoding, adding special tokens, etc:
 
 ```bash
-python prepare_data.py --data_dir data/toy \
---max_seq_length=128 \
---output_dir=data/toy \ 
---pretrained_model_name=117M
+python prepare_data.py --data-dir data/toy \
+    --max-seq-length=128 \
+    --output-dir=data/toy \
+    --pretrained-model-name=117M
 ```
 
-- `data_dir`: The directory of raw data, wherein data files must be named as 'train.txt', 'dev.txt', or 'test.txt'. It is *not* necessary to provide all three files.
-- `max_seq_length`: The maxium length of sequence after BPE encoding. This includes GPT-2 special tokens that will be automatically added. Longer sequence will be trimmed. 
-- `output_dir`: The output path where the resulting pickle files will be put in. Be default, it is set to be the same as `data_dir`. 
-- `pretrained_model_name`: The name of a pre-trained model to load selected in the list of: `117M`, `345M`.
+- `--data-dir`: The directory of raw data, wherein data files must be named as 'train.txt', 'dev.txt', or 'test.txt'. It
+  is *not* necessary to provide all three files.
+- `--max-seq-length`: The maximum length of sequence after BPE encoding. This includes GPT-2 special tokens that will be
+  automatically added. Longer sequence will be trimmed. 
+- `--output-dir`: The output path where the resulting pickle files will be put in. Be default, it is set to be the same
+  as `--data-dir`. 
+- `--pretrained_model_name`: The name of a pre-trained model to load selected in the list of: `117M`, `345M`.
 
-The above cmd will output pickle files in the specified output directory. E.g., if `train.txt` is provided under `data_dir`, the output file `train.pickle` will be produced under `output_dir`.
+The above cmd will output pickle files in the specified output directory. E.g., if `train.txt` is provided under
+`data_dir`, the output file `train.pickle` will be produced under `output_dir`.
```

Comment:
I think these are actually fine. If you take a look at the help output given by `argparse`, it will look like:
```
In [1]: import argparse

In [2]: parser = argparse.ArgumentParser()

In [3]: parser.add_argument("--some-arg")
Out[3]: _StoreAction(option_strings=['--some-arg'], dest='some_arg', nargs=None, const=None, default=None, type=None, choices=None, help=None, metavar=None)

In [4]: parser.parse_args(["--help"])
usage: ipython [-h] [--some-arg SOME_ARG]

optional arguments:
  -h, --help           show this help message and exit
  --some-arg SOME_ARG

SystemExit: 0
```
where the value of the argument `--some-arg` is referred to as `SOME_ARG`. I think using a lowercased version is also fine.